.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,29 @@
-Thanks for taking the time to report a bug in the Origami project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/Origami/issues)._
+Thanks for taking the time to report a bug in the Origami project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/Origami/issues).
+For more information, see the [CONTRIBUTING Guide](https://github.com/Cloud-CV/Origami/blob/master/.github/CONTRIBUTING.md) 
+
+## Description
+
+[Description of the bug or feature]</br>
```

Comment:
Without a <br/> here would be better. We should include html as less as possible.

Reply:
:+1: